docs/anatomy/anatomy-repository.md

```diff
@@ -13,10 +13,11 @@ Repositories do not have any requirements to be compatible with KintoHub. You ca
 ## Setting New Repository
 
 1. Login to KintoHub
-2. Click **Create Service** at the top right
-3. Enter in your **Git Repository URL**
-4. Enter in your **Branch** you wish to build and deploy
-5. Optionally add an **Access Token** to your repository if it is private.
+2. Click on the **Create Service** button at the top right
+3. Click on the **Import URL** tab
+4. Enter in your **Git Repository URL**
+5. Enter in your **Branch** you wish to build and deploy
+6. Optionally add an **Access Token** to your repository if it is private.
 
 ## Edit Existing Repo
```

Comment:
action - in this repository thing. Maybe at the top of the page after describing the feature, "Looking to connect your repoisotires via Github Apps. Read more here. (link to github apps stuff)

Reply:
Added a link to Github Apps.